selfdrive/car/__init__.py

```diff
@@ -261,14 +261,15 @@ def override(self, **kwargs):
 
 @dataclass(order=True)
 class PlatformConfig(Freezable):
-  platform_str: str
   car_info: list[CarDocs]
   specs: CarSpecs
 
   dbc_dict: DbcDict
 
   flags: int = 0
 
+  platform_str: str | None = None
```

Comment:
why can this be None now?

Reply:
platform_str is None when PlatformConfig is not in enum.

For example:

```python
config = ChryslerPlatformConfig(
  [ChryslerCarDocs("Ram 1500 2019-24", car_parts=CarParts.common([CarHarness.ram]))],
  ChryslerCarSpecs(mass=2493., wheelbase=3.88, steerRatio=16.3, minSteerSpeed=14.5)
 )
 assert config.platform_str == None
 ```

Reply:
can it just raise an exception instead?

Reply:
> can it just raise an exception instead?

i can try to implement this by overriding `getattr` in metaclass, but i'm not sure is it worth it.